sdk/storage/storage-queue/src/QueueClient.ts

```diff
@@ -221,12 +225,16 @@ export class QueueClient extends StorageClient {
    *                     "https://myaccount.queue.core.windows.net/myqueue". You can
    *                     append a SAS if using AnonymousCredential, such as
    *                     "https://myaccount.queue.core.windows.net/myqueue?sasString".
-   * @param {Credential} credential Such as AnonymousCredential, SharedKeyCredential or TokenCredential.
+   * @param {Credential | TokenCredential } credential Such as AnonymousCredential, SharedKeyCredential or TokenCredential.
```

Comment:
>TokenCredential [](start = 107, length = 15)

Such as AnonymousCredential, SharedKeyCredential RawTokenCredential or TokenCredentials retrieved from @azure/identity package

Reply:
I'll update all of the comments across the various Client files

Reply:
need updates in samples folder and readme.md